resotocore/resotocore/config/config_handler_service.py

```diff
@@ -66,11 +78,49 @@ async def coerce_and_check_model(self, cfg_id: ConfigId, config: Json, validate:
         # If we come here, everything is fine
         return final_config
 
+    async def coerce_config(self, config: Json) -> Json:
```

Comment:
why does `coerce_and_check_model` does not use `coerce_config` - please DRY it up.

Reply:
I'm not sure that would help because in coerce_and_check_model coercion is done implicitly via the `check_valid` method, and DRYing it would make the whole thing less readable.

Reply:
Missed that point - makes sense.